test/lit.cfg

```diff
@@ -135,7 +135,8 @@ if len(kleaver_extra_params) != 0:
 # Set absolute paths and extra cmdline args for KLEE's tools
 subs = [ ('%kleaver', 'kleaver', kleaver_extra_params),
   ('%klee','klee', klee_extra_params),
-  ('%ktest-tool', 'ktest-tool', '')
+  ('%ktest-tool', 'ktest-tool', ''),
+  ('%klee-replay', 'klee-replay', '')
```

Comment:
Ah now I remember. I've seen this before.

`%klee-replay` is expanding to **`/path/to/build/bin/klee`-replay (i.e. `%klee` substitution is being used rather than `%klee-replay`) for the non meta-SMT builds we don't set `klee_extra_params` 
so this happens to work by accident. When we set `klee_extra_params` though it will break because then the expansion is

**`/path/to/build/bin/klee -some-option`**`-replay`

Changing the order as I suggest should fix this. Alternatively using a substitution name that does have `klee` as  a prefix is also an option.